prisma/schema.prisma

```diff
@@ -267,8 +283,19 @@ model GeneratedK1 {
 // Hosted Image
 // -----------------
 model HostedImage {
-  id        String   @id
-  filename  String
-  createdAt DateTime @default(now())
-  is_used   Boolean  @default(false)
+  id                Int      @id @default(autoincrement())
+  filename          String
+  provider_image_id String
+  provider          String
+  url               String
+  createdAt         DateTime @default(now())
+  is_used           Boolean  @default(false)
+
+  ProjectThumbnail    Project[]   @relation("ProjectThumbnail")
+  ProjectCoverImage   Project[]   @relation("ProjectCoverImage")
+  CategoryCoverImage  Category[]  @relation("CategoryCoverImage")
+  AwardImage          Award[]     @relation("AwardImage")
+  HackathonCoverImage Hackathon[] @relation("HackathonCoverImage")
+  StoryCoverImage     Story[]     @relation("StoryCoverImage")
```

Comment:
This might not be a very important point, but in general we are using a naming conventions for the fields in a table so that:
ProjectCoverImange => project_cover_image

Just to keep things consistent throughout the codebase